pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -130,6 +130,28 @@ type WorkerPool struct {
 	// Zones contains information about availability zones for this worker pool.
 	// +optional
 	Zones []string `json:"zones,omitempty"`
+	// Settings contains configurations for different worker-pools. Eg. MachineDrainTimeout, MachineHealthTimeout.
+	// +optional
+	Settings *Settings `json:"settings,omitempty"`
```

Comment:
Why don't you simply reuse the `gardencorev1beta1` struct here?

Reply:
Yes, that's a good idea, I'll reuse the struct here.

Reply:
done. ref: https://github.com/gardener/gardener/commit/9aacdeb0348a4d06173bb54d71af652807da0846#diff-5865efa9645384006d2ea212587c1ce4R135